teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -1608,6 +1608,10 @@ private open class TestOccurrenceImpl(bean: TestOccurrenceBean): TestOccurrence
 
     override val buildId: BuildId = BuildId(bean.build!!.id!!)
 
+    override val nextFixed: BuildId = BuildId(bean.nextFixed?.id ?: "")
```

Comment:
What's the sense to have BuildId with empty string inside? Maybe it would be better to return `null` instead to clearly indicate that there is no such build?

Reply:
Please, check the new implementation.

Reply:
Thanks. There is the same problem with `firstFailed` though.

Reply:
Oh my :/ fixed now